lib/interactive_viewer.ml

```diff
@@ -60,9 +60,40 @@ let navigate z_patches (dir : direction) : unit =
   | Prev -> Lwd.set z_patches (Zipper.prev z)
   | Next -> Lwd.set z_patches (Zipper.next z)
 
-(* let pure_str s = Lwd.pure (W.string s) *)
 let quit = Lwd.var false
 
+let add_lines lines =
+  let add_line (addition, removal) line =
+    match line with
+    | `Their _ -> (addition + 1, removal)
+    | `Mine _ -> (addition, removal + 1)
+    | `Common _ -> (addition, removal)
+  in
+  List.fold_left add_line (0, 0) lines
+
+let add_hunks hunks =
+  List.fold_left
+    (fun total_counts hunk ->
+      let additions, removals = add_lines hunk.Patch.lines in
+      (fun (add1, remove1) (add2, remove2) -> (add1 + add2, remove1 + remove2))
+        total_counts (additions, removals))
```

Comment:
You are creating an anonymous function that you are directly applying to some arguments.

This example is more obvious but very similar to yours:
```
(fun x y -> x + y) 4 10
```
is a convoluted version of
```
4 + 10
```

Could you change this similarly to avoid the unnecessary function?